CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # CHANGELOG
 
+
+## Version v0.2.0
```

Comment:
Good catch. Updated to 0.3.0. Let me know if there's any reason we should keep the alpha, but since this is now branched off a clean main, it should be a normal release.